pam_tcb/support.h

```diff
@@ -178,4 +178,13 @@ extern int unix_getspnam(struct spwd **, const struct passwd *);
 extern char *crypt_wrapper(pam_handle_t *, const char *, const char *);
 extern char *do_crypt(pam_handle_t *, const char *);
 
+/* Helper function around getlogin() */
+static inline char *pam_tcb_getlogin(void)
+{
+	char *login = getlogin();
+	if (!login)
+	    return "";
+	return login;
+}
```

Comment:
The indentation in `pam_tcb_getlogin` is inconsistent with what we use in this project (we use tabs, and at least I assumed they were 8-char wide if this ever matters).

Reply:
Fixed in rebased commit.

Reply:
Maybe it's just me, but at least the way GitHub displays these changes to me they still have inconsistent first-level (function level) indentation compared e.g. to other functions we have in tcb.

Reply:
If you check the [diff output](https://github.com/openwall/tcb/pull/8.patch), you can verify the indentation is fully correct.

Reply:
I usually fetch PRs and look at them using git diff, and I confirm this one is indented properly.